test/e2etest/test-config.tfvars.tmpl

```diff
@@ -31,6 +31,11 @@ instance_types = ["m5.2xlarge"]
 # Domain name base for the ingress controller. The final domain is subdomain within this domain. (eg.: environment.domain.com)
 {{if .domain}}domain = "{{.domain}}"{{end}}
 
+# Enable osquery in cluster worker nodes. Secret dcd-ci-k8s-fleet_enrollment_secret20211025115816661700000003-DOGYSr
+# is replicated in all regions in 887764444972 account. If new regions are enabled, the secret should be
+# replicated to these regions as well
+osquery_fleet_enrollment_secret_name = "dcd-ci-k8s-fleet_enrollment_secret20211025115816661700000003-DOGYSr"
```

Comment:
we may better to read this from environment variable instead of hard coding it here

Reply:
Good idea. Done